src/components/spotlight/spotlight.html.twig

```diff
@@ -0,0 +1,146 @@
+    - "poster" (string) (default: ''): Image to be used as the video placeholder 
+    - "sources" (array) (default: []): format: [
+      {
+        "src" (string) (default: ''),
+        "type" (string) (default: ''),
+      },
+      ...
+    ],
+    - "tracks" (array) (default: []): format: [
+      {
+        "src" (string) (default: ''),
+        "kind" (string) (default: ''),
+        "src_lang" (string) (default: ''),
+        "label" (string) (default: ''),
+      },
+      ...
+    ],
+  - "sr_play" (string) (default: '') screen reader label for the play button
+  - "sr_pause" (string) (default: '') screen reader label for the pause button
+  - "credit" (string) (default: '') Credit for the image
+  - "full_width" (boolean) (default: false) Full width spotlight (extend outside the grid)
+  - "extra_classes" (string) (default: '')
+  - "extra_attributes" (optional) (array) (default: [])
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (optional) (string) Attribute value, eg: 'data-test-1'
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-spotlight' %}
+{% set _extra_attributes = '' %}
+{% set _color_mode = color_mode|default('') %}
+{% set _picture = picture|default({}) %}
+{% set _video = video|default({}) %}
+{% set _font_size = font_size|default('m') %}
+{% set _full_width = full_width|default(false) %}
+{% set _header = header|default('') %}
+{% set _title = title|default('') %}
+{% set _path = path|default('') %}
+{% set _credit = credit|default('') %}
+{% set _has_anchor = has_anchor|default(true) %}
+{% set _sr_play = sr_play|default('') %}
+{% set _sr_pause = sr_pause|default('') %}
+
+{# Internal logic - Process properties #}
+
+{% if _color_mode is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ 'ecl-color-mode--' ~ _color_mode %}
+{% endif %}
+
+{% if _font_size is not empty %}
+  {% set _css_class = _css_class ~ ' ecl-spotlight--font-' ~ _font_size %}
+{% endif %}
+
+{% if _full_width %}
+  {% set _css_class = _css_class ~ ' ecl-spotlight--full-width' %}
+{% endif %}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% if attr.value is defined %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name|e('html_attr') ~ '="' ~ attr.value|e('html_attr') ~ '"' %}
+    {% else %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name|e('html_attr') %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<section
+  class="{{ _css_class }}"
+  {{ _extra_attributes|raw }}
+>
+  {# Header #}
+  {% if _header is not empty %}
+    <div class="ecl-spotlight__header">
+      <div class="ecl-container">
+        {{- _header -}}
+      </div>
+    </div>
+  {% endif %}
+
+  {# Link #}
+  {% if _path is not empty %}
```

Comment:
is this really how it is intended..? Fully clickable but not on the "header..? Anyway, it's just a personal feeling, but i don't like the idea of an <a> tag wrapping the whole thing, i would prefer defining it as an element and then extend its area to the full width and height needed, probably, but it's just a preference, you can ignore it.